Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -3567,6 +3569,106 @@ NonnullRefPtr<StyleValue const> StyleComputer::compute_opacity(NonnullRefPtr<Sty
     VERIFY_NOT_REACHED();
 }
 
+// https://drafts.csswg.org/css-anchor-position/#position-area-computed
+NonnullRefPtr<StyleValue const> StyleComputer::compute_position_area(NonnullRefPtr<StyleValue const> const& specified_value)
+{
+    // The computed value of a <position-area> value is the two keywords indicating the selected tracks in each axis,
+    // with the long (block-start) and short (start) logical keywords treated as equivalent. It serializes in the order
+    // given in the grammar (above), with the logical keywords serialized in their short forms (e.g. start start
+    // instead of block-start inline-start).
+    if (specified_value->is_keyword())
+        return specified_value;
+
+    auto to_short_keyword = [](NonnullRefPtr<KeywordStyleValue const> const& keyword_value) -> NonnullRefPtr<KeywordStyleValue const> {
+        switch (keyword_value->keyword()) {
+        case Keyword::BlockStart:
+        case Keyword::InlineStart:
+            return KeywordStyleValue::create(Keyword::Start);
+        case Keyword::BlockEnd:
+        case Keyword::InlineEnd:
+            return KeywordStyleValue::create(Keyword::End);
+        case Keyword::SelfBlockStart:
+        case Keyword::SelfInlineStart:
+            return KeywordStyleValue::create(Keyword::SelfStart);
+        case Keyword::SelfBlockEnd:
+        case Keyword::SelfInlineEnd:
+            return KeywordStyleValue::create(Keyword::SelfEnd);
+        case Keyword::SpanBlockStart:
+        case Keyword::SpanInlineStart:
+            return KeywordStyleValue::create(Keyword::SpanStart);
+        case Keyword::SpanBlockEnd:
+        case Keyword::SpanInlineEnd:
+            return KeywordStyleValue::create(Keyword::SpanEnd);
+        case Keyword::SpanSelfBlockStart:
+        case Keyword::SpanSelfInlineStart:
+            return KeywordStyleValue::create(Keyword::SpanSelfStart);
+        case Keyword::SpanSelfBlockEnd:
+        case Keyword::SpanSelfInlineEnd:
+            return KeywordStyleValue::create(Keyword::SpanSelfEnd);
+        default:
+            break;
+        }
+        return keyword_value;
+    };
+
+    VERIFY(specified_value->is_value_list());
```

Comment:
This `VERIFY` happens in `->as_value_list()` as well; imo we should just use the `as_...()` methods directly and depend on them to check their type.